dae/dae/gpf_instance/gpf_instance.py

```diff
@@ -27,40 +27,106 @@ def __init__(self, config_file='DAE.conf', work_dir=None, defaults=None):
         self.dae_config = DAEConfigParser.read_and_parse_file_configuration(
             config_file=config_file, work_dir=work_dir, defaults=defaults
         )
-
-        self.genomes_db = GenomesDB(
-            self.dae_config.dae_data_dir,
-            self.dae_config.genomes_db.conf_file
-        )
-
-        self.pheno_factory = PhenoFactory(dae_config=self.dae_config)
-
-        self.gene_info_config = \
-            GeneInfoConfigParser.read_and_parse_file_configuration(
-                self.dae_config.gene_info_db.conf_file,
-                self.dae_config.dae_data_dir
+        self.genomes_db_ = None
```

Comment:
To make something protected the `_` is position in the front of variable. To mark it as private is used `__`.